internal/pkg/service/buffer/store/model/mapper/mapper.go

```diff
@@ -0,0 +1,204 @@
+package mapper
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/c2h5oh/datasize"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/idgenerator"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/api/gen/buffer"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/store/key"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/store/model"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/store/model/column"
+	serviceError "github.com/keboola/keboola-as-code/internal/pkg/service/common/errors"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/errors"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/strhelper"
+)
+
+func ReceiverPayloadFromModel(bufferAPIHost string, model model.Receiver) buffer.Receiver {
```

Comment:
Note: ak by si ten mapper spravil ako strukturu, a funkcie ako metody, tak by si tu nemusel predavat parameter `bufferAPIHost` (a ziskavat ho v service), ak bude takych veci viac, tak by som to zvazil.